### PR TITLE
Update RuleLocationCollection.php - sort order and breadcrumb trail

### PR DESCRIPTION
**Pull request by SylvieCoz**
- Order Location Rules by name rather than ID
- Add this rule collection to the breadcrumb trail

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #number

### CODE REVIEW

File: src/RuleLocationCollection.php [hunk 1]
```diff
@@ -39,7 +39,9 @@ class RuleLocationCollection extends RuleCollection
 {
     public $stop_on_first_match = true;
     public static $rightname    = 'rule_location';
-
+    public $orderby             = 'name';
```
Comment (cedric-anne) [inline src/RuleLocationCollection.php:42]: I do not know why it is still possible to change the field used to define rules display order, but it completely breaks the ability to sort them. Indeed, while it is still possible to use the drag&drop feature (see image), their ranking is changed but they are always loaded and evaluated based on a name sorting (and reloading the page always show them ordered by name)
image

```suggestion
```

File: src/RuleLocationCollection.php [hunk 2]
```diff
@@ -39,7 +39,9 @@ class RuleLocationCollection extends RuleCollection
...
+    public $menu_option         = 'location';
+
```
Comment (cedric-anne) [inline src/RuleLocationCollection.php:44]:
```suggestion

```

File: src/RuleLocationCollection.php (hunk 1 above)
Comment (SylvieCoz) [reply src/RuleLocationCollection.php:42]: Cédric, you're right : I missed the drag&drop feature, so forget the adding of **$orderby** suggestion
FYI, I took the **$orderby** instruction from the RuleRightCollection class (need to be corrected as well ?)

I still suggest the adding of **$menu_option** to have the breadcrumb trail updated (see image)
image

**Review by cedric-anne — APPROVED** [review]

**Review by trasher — APPROVED** [review]

**Comment by cedric-anne** [comment]
Thanks for the contribution.

File: src/RuleLocationCollection.php (hunk 1 above)
Comment (cedric-anne) [reply src/RuleLocationCollection.php]: > FYI, I took the **$orderby** instruction from the RuleRightCollection class (need to be corrected as well ?)

#13988 will remove ability to override default order field.

**Pull request merged**